src/librepgp/stream-common.cpp

```diff
@@ -910,9 +925,22 @@ init_tmpfile_dest(pgp_dest_t *dst, const char *path, bool overwrite)
         RNP_LOG("failed to build file path");
         return RNP_ERROR_BAD_PARAMETERS;
     }
+#ifdef HAVE_MKSTEMP
+    int fd = mkstemp(tmp);
+    if (fd < 0) {
+        RNP_LOG("failed to create temporary file with tempate '%s'. Error %d.", tmp, errno);
+        return RNP_ERROR_WRITE;
+    }
+#else
```

Comment:
Do I correctly understand that part after `#else` is supposed to run only under Windows? Then it would be cleaner to use windows-related define, with additional `#else #error Unsupported os` afterwards.

Reply:
yes, I assumed `mkstemp` is available on all other platforms